_posts/2015-11-16-tips-for-a-friendly-open-source-experience.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+name: Tips-for-a-friendly-open-source-experience
+title: 'Tips for a friendly open source experience'
+date: 2015-11-16 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/friendly.png
+tags:
+- opensource
+- oss
+- github
+
+---
+
+Open source has moved from being an important actor in software development to being central for plenty of people, especially thanks to [Github](https://github.com). Most of us are simple users of OSS (Open-source software) but being brave and taking a step forward and becoming a contributor, or even an owner, could help you massively in different ways:
+
+* Collaborating with clever people disregarding of place of residence.
```

Comment:
> ...clever people disregarding of place of residence.

I missed this one but i think it could read better

> ...clever people irrespective of their place of residence.